pkg/users/accounts-list.js

```diff
@@ -84,29 +125,54 @@ const UserActions = ({ account }) => {
     return kebab;
 };
 
-const getAccountRow = (account, current, groups) => {
-    const adminGroups = ['sudo', 'root', 'wheel'];
-
-    const userGroups = groups.reduce((pV, group) => {
-        if (group.userlist.find(accountName => accountName === account.name)) {
-            const isAdmin = !!adminGroups.find(adm => adm === group.name);
-            return pV.concat({ name: group.name, members: group.userlist.length, isAdmin: isAdmin });
-        } else {
-            return pV;
-        }
-    }, []);
-
-    userGroups.sort((a, b) => {
-        if (a.isAdmin)
-            return -1;
-        if (b.isAdmin)
-            return 1;
-        if (a.members === b.members)
-            return a.name.localeCompare(b.name);
-        else
-            return b.members - a.members;
-    });
+const getGroupRow = (group, accounts) => {
+    let groupColorClass;
+    if (group.isAdmin)
+        groupColorClass = "group-gold";
+    else if (group.members > 0)
+        groupColorClass = "group-cyan";
+    else
+        groupColorClass = "group-grey";
 
+    const columns = [
+        {
+            title: <Flex alignItems={{ default: 'alignItemsCenter' }}><div className={"dot " + groupColorClass} /><FlexItem>{group.name}</FlexItem></Flex>,
```

Comment:
You'll need to use some `sortKey` here. Right now the order when sorting by group name is not alphabetical and it changes on every second click

Reply:
Fixed